packages/fxa-auth-server/config/index.ts

```diff
@@ -963,6 +963,12 @@ const conf = convict({
         default: true,
       },
     },
+    googleMapsApiKey: {
```

Comment:
Hey Reino; I chatted with Ben about this a little, and he mentioned that this is general purpose API that happens to be used by the subscriptions component, so it'd be better not to put the API key config under the `subscriptions` key in config. Can you move it?

E.g. we may extend this Google Maps service in the future, and it could be used by other areas of the fxa-auth-server code.